cmd/ethrex/initializers.rs

```diff
@@ -405,6 +405,15 @@ pub async fn init_l1(
         store.generate_flatkeyvalue()?;
     }
 
+    if let Some(latest_store_schema_version) = store.get_store_schema_version().await?
+        && latest_store_schema_version != STORE_SCHEMA_VERSION
```

Comment:
This should fail if the file is not found

Reply:
Right. But if we are using an old ethrex without DB versioning, this would pass, right?